packages/translations/scripts/__tests__/extract-string.spec.js

```diff
@@ -1,4 +1,4 @@
-const fs = require('fs');
+// const fs = require('fs');
```

Comment:
if we don't need this please remove it instead of commenting

Reply:
I'm not sure about this. they commented some parts of the code but they forgot to comment this variable

Reply:
just remove it. anyone wants to use it will import it.

Reply:
sure ako